packages/db/package.json

```diff
@@ -23,6 +23,7 @@
         "@lucia-auth/adapter-drizzle": "^1.0.7",
         "@planetscale/database": "^1.18.0",
         "drizzle-orm": "^0.31.2",
+        "mathjs": "^13.0.2",
```

Comment:
Wird nur auf dem Server verwendet oder?
Dann können wir das noch zu den externalPackages in den cnfigs hinzufügen um zu vermeiden, dass es im Bundle landet.
Sind sonst nämlich 180KB.